handlers/messageReactionRemove.js

```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line no-unused-vars
+const { MessageReaction, User } = require("discord.js");
+const ReactionRoleManager = require("../classes/ReactionRoleManager");
+
+/**
+ * @function
+ * @param {MessageReaction} messageReaction
+ * @param {User} user
+ * @returns {Promise<void>}
+ */
+module.exports = async (messageReaction, user) => {
+  const manager = new ReactionRoleManager(messageReaction, user);
```

Comment:
can we add a new config setting to enable/disable this behavior, maybe something like `removeRolesOnReactionRemove`

the line immediately following creating the manager can call something like:
```suggestion
  const manager = new ReactionRoleManager(messageReaction, user);
  if (!manager.handleRolesOnReactionRemove()) {
    return;
  }
```

README will also have to update, instruction text can be something like:

- `removeRolesOnReactionRemove` determines whether a user's previously assigned roles are removed when the user removes their reaction (`true`) or not (`false`). Note that `removeReaction` must be `false` for this to work properly.

Reply:
copy paste for your convenience

```
- `removeRolesOnReactionRemove` determines whether a user's previously assigned roles are removed when the user removes their reaction (`true`) or not (`false`). Note that `removeReaction` must be `false` for this to work properly.
```